src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -117,7 +119,9 @@ internal override bool ProcessAssemblyDesigner (AssemblyDefinition assembly)
 					if (output.ContainsKey (key)) {
 						LogMessage ($"          Found duplicate {key}");
 					} else {
+						LogMessage ($"          Adding {key}");
 						output.Add (key, property.GetMethod);
+						caseInsensitiveLookup.Add (key, property.GetMethod);
```

Comment:
This could be a problem, as `.Add()` will throw if a "duplicate key" already exists:

```csharp
var d = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
d.Add ("A", "a");
d.Add ("a", "a");                                                                                                                     
// throws System.ArgumentException: An item with the same key has already been added. Key: a                                                              ```

Either we should use `caseInsensitiveLookup.TryAdd(key, property.GetMethod)` (.NET Standard 2.1), or we should use the indexer `caseInsensitiveLookup[key] = property.GetMethod`, though using the indexer means that `caseInsensitiveLookup` will always contain the *last* case-insensitive value encountered rather than the first (which might be fine?).

Reply:
well we only use .netstandard 2.0 so I guess that makes the decision for us.